types/underscore/index.d.ts

```diff
@@ -1013,6 +1013,13 @@ declare namespace _ {
             step?: number,
         ): number[];
 
+        /**
+         * Internal wrapper for `_.toPath` to enable minification.
+         * @param path Incoming Path.
+         * @returns an array containing the names of each property on the path from the root object to the target property.
+         */
+        toPath(path:any): any[];
```

Comment:
I think `path` by default is `number | string | (number | string)[]`. The return value must be `(number | string)[]`.

Reply:
Are these the return values of the toPath (path:any) function? number | string | (number | string)[](number | string)[]

Reply:
I mean the type should be like this:

```suggestion
        toPath(path: number | string | (number | string)[]): (number | string)[];
```